### PR TITLE
Setup automatic build

### PR DESCRIPTION
**Pull request by satyaranjanp**
Perform verification of PRs raised to this repo.

### CODE REVIEW

File: .github/workflows/ci-build.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:bionic
+    env:
+      repo: connection-limit
+
+    steps:          
+      - name: Setup build environment
```
Comment (dthaler) [inline .github/workflows/ci-build.yaml:17]:
```suggestion
      - name: Set up build environment
```
nit: "set up" is the verb, "setup" is a noun

File: .github/workflows/ci-build.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          libssl-dev \
+          linux-headers-generic \
+          linux-tools-common \
+          linux-tools-generic \
+          llvm \
+          rsync
+      
+      - name: Clone dependencies 
+        run: |
+          chmod 777 /usr
```
Comment (dthaler) [inline .github/workflows/ci-build.yaml:46]: Why do you need this line?  This seems bad practice to me.

File: .github/workflows/ci-build.yaml (hunk 1 above)
Comment (satyaranjanp) [reply .github/workflows/ci-build.yaml]: Done.

File: .github/workflows/ci-build.yaml (hunk 2 above)
Comment (satyaranjanp) [reply .github/workflows/ci-build.yaml]: Removed this dependency.

**Review by dthaler — APPROVED** [review]

**Review by sanfern — APPROVED** [review]

**Review by KanthiPavuluri — APPROVED** [review]
LGTM

**Pull request merged**